src/main.go

```diff
@@ -72,6 +84,8 @@ func main() {
 			fmt.Print("\033[J\033[2;0H\r  ")
 			fmt.Printf("ID: %d | NextID: %d | NextAddr: %s ", elevState.NodeID, elevState.NextNode.ID, elevState.NextNode.Addr)
 
+		case message := <-incomingMessageChannel:
```

Comment:
I like adding an empty line under the cases in the switch statement to separate the blocks a bit.